src/pages/dashboard/listings/index.tsx

```diff
@@ -615,21 +615,24 @@ function Bounties() {
                               <></>
                             )}
 
-                            <MenuItem
-                              py={2}
-                              color={'brand.slate.500'}
-                              fontSize={'sm'}
-                              fontWeight={500}
-                              icon={<CopyIcon h={4} w={4} />}
-                              onClick={() =>
-                                window.open(
-                                  `${router.basePath}/dashboard/listings/${currentBounty.slug}/duplicate`,
-                                  '_blank',
-                                )
-                              }
-                            >
-                              Duplicate
-                            </MenuItem>
+                            {(currentBounty.type === 'bounty' ||
```

Comment:
the changes to this file is not from your commit; can you remove this please?

Reply:
Done!